app/views/cards/show.html.erb

```diff
@@ -48,7 +27,28 @@
           <div class="tab-pane active" id="info-tab">
             <% cache ['card-information-tab', @card] do %>
               <div class="note-text-inner">
-                <h4 class="mb-4 header-underline"><%= @card.name %></h4>
+                <h4 class="mb-4 header-underline">
+                  <span class="text-truncate" title="<%= @card.name %>"><%= @card.name %></span>
```

Comment:
Is `text-truncate` a new change? Why are we truncating instead of using `word-wrap: break-word;` if it's overflowing?

Reply:
we are limiting the header to a single line to have consistent headers with the dots menu always in the same position. If we wrap the words instead, the header can be infinitely tall creating an ugly view.

Reply:
Also, if the text is truncated, the user can still see the full version in a tooltip when they hover over the header.